packages/mdc-select/adapter.ts

```diff
@@ -90,47 +85,70 @@ export interface MDCSelectAdapter {
   closeOutline(): void;
 
   /**
-   * Opens the menu.
+   * Sets the select to disabled.
    */
-  openMenu(): void;
+  setDisabled(isDisabled: boolean): void;
 
   /**
-   * Closes the menu.
+   * Sets the line ripple transform origin center.
    */
-  closeMenu(): void;
+  setRippleCenter(normalizedX: number): void;
 
   /**
-   * Returns true if the menu is currently open.
+   * Emits a change event when an element is selected.
    */
-  isMenuOpen(): boolean;
+  notifyChange(value: string): void;
 
   /**
-   * Sets the selected index of the select to the index provided.
+   * Checks if the select is currently valid.
    */
-  setSelectedIndex(index: number): void;
+  checkValidity(): boolean;
 
   /**
-   * Sets the select to disabled.
+   * Adds/Removes the invalid class.
    */
-  setDisabled(isDisabled: boolean): void;
+  setValid(isValid: boolean): void;
 
   /**
-   * Sets the line ripple transform origin center.
+   * Sets the text content of the selectedText element to the given string.
    */
-  setRippleCenter(normalizedX: number): void;
+  setSelectedText(text: string): void;
 
+  // Menu-related methods ======================================================
   /**
-   * Emits a change event when an element is selected.
+   * Opens the menu.
    */
-  notifyChange(value: string): void;
+  openMenu(): void;
 
   /**
-   * Checks if the select is currently valid.
+   * Closes the menu.
    */
-  checkValidity(): boolean;
+  closeMenu(): void;
 
   /**
-   * Adds/Removes the invalid class.
+   * Returns true if the menu is currently open.
    */
-  setValid(isValid: boolean): void;
+  isMenuOpen(): boolean;
+
+  /**
+   * Sets the attribute on the menu item at the given index. If `attributeValue`
+   * is undefined, removes the attribute.
+   */
+  setAttributeAtIndex(index: number, attributeName: string, attributeValue?: string): void;
```

Comment:
To make this explicite, do you think we need to split this into two separate adapter methods for set and remove?

It'll be consistant with other adapter methods.